.github/workflows/first-interaction.yml

```diff
@@ -0,0 +1,21 @@
+name: First Interaction
+
+on:
+  issues:
+    types: [ opened ]
+  pull_request_target:
+    types: [ opened ]
+
+jobs:
+  add-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: 'Thanks for your contribution :fire: We will take a look asap :rocket:'
+          pr-message: >-
+            We are always happy to welcome new contributors :heart: To make things easier for everyone, please
+            make sure to follow our [contribution guidelines](https://github.com/eclipse-edc/Connector/blob/main/CONTRIBUTING.md),
```

Comment:
this should probably point to this repo's `Contributing.md`, which in turn could point to EDC's `Contributing.md`.